### PR TITLE
whisper large evaluation code

### PR DESCRIPTION
**Pull request by yintong-lu**
## Type of Change

example
add quantization example
dataset libraryspeech about 20G
please validate@chensuyue 
baseline is provided by @yintong-lu 
fp32 wer: 3.042%
dynamic quant int8 wer: 3.645%
copy cache to dataset machine.

## Description

detail description: add example of whisper-large speech recognition (evaluation code)

## Expected Behavior & Potential Risk

no expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

no library dependency introduced or removed

### CODE REVIEW

**Review by xin3he — APPROVED** [review]

File: examples/pytorch/speech_recognition/whisper_large/quantization/ptq_static/fx/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Prerequisite
+
+### 1. Environment
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+```shell
+cd examples/pytorch/speech_recognition/whisper_large/quantization/ptq_static/fx
+pip install -r requirements.txt
```
Comment (chensuyue) [inline examples/pytorch/speech_recognition/whisper_large/quantization/ptq_static/fx/README.md:15]: Please add FWs related components into the requirements.txt

File: examples/pytorch/speech_recognition/whisper_large/run_whisper_large.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from datasets import load_dataset
```
Comment (chensuyue) [inline examples/pytorch/speech_recognition/whisper_large/run_whisper_large.py:1]: Do we need keep this file?

**Comment by chensuyue** [comment]
[extension test for whisper_large] (https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4875/)

**Comment by changwangss** [comment]
https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4900/

**Comment by changwangss** [comment]
accuracy: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4903 benchmark "args.warmup" is wrong name, fixed.
benchmark: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4905

**Review by changwangss — APPROVED** [review]

**Comment by hshen14** [comment]
LGTM. Is the script good to support other size like small, tiny?

**Review by xin3he — APPROVED** [review]

**Comment by changwangss** [comment]
> LGTM. Is the script good to support other size like small, tiny?

sure, but I am not sure if the int8 model can reach the relative 1% accuracy drop.

**Pull request merged**